01-backup.md

```diff
@@ -54,6 +54,23 @@ we're telling Git:
 
 The four commands above only need to be run once:
 the flag `--global` tells Git to use the settings for every project on this machine.
+The configuration options get stored in your home directory in a text file called `.gitconfig`.
+
+~~~ {.bash}
+$ cat ~/.gitconfig
+~~~
+~~~ {.output}
+[user]
+	name = Vlad Dracula
+	email = vlad@tran.sylvan.ia
+[color]
+	ui = true
+[core]
+	editor = nano
+~~~
+
+This file can be edited directly.
```

Comment:
> I frequently edit mine by hand ;).

Some times I do it too ;). I just want avoiding breaking it enough to cause confusion.